.circleci/config.yml

```diff
@@ -120,8 +155,29 @@ jobs:
           paths:
             - ~/spree/vendor/bundle
 
+  bundle_ruby_3_1:
+    <<: *defaults_3_1
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - spree-bundle-v10-ruby-3-1-{{ .Branch }}
+            - spree-bundle-v10-ruby-3-1
+      - run:
+          name: Install libvips
+          command: sudo apt-get update && sudo apt-get install libvips42
```

Comment:
Curious why you are installing `libvips42` instead of normal `libvips` for ruby 3.1 build? I did a quick google search but couldn't tell what the difference was between the two.

Reply:
I changed the image from legacy `circleci/ruby` to `cimg/ruby` and the linux there has the same package with a different name ;)

Reply:
Aha, I see :D